content/dev_stories/playground_to_production_fruit_falling.md

```diff
@@ -0,0 +1,677 @@
+and about how long I could expect it take.
+
+1.	On my GitHub account, I created a new repository from the 
+[Babylon.js NPM Package Template](https://github.com/BabylonJS/npm-package-template), 
+moved my Playground code into that repo, and confirmed it worked. **Within 
+half an hour, I had my Playground code running in my own repo.**
+1.	I then created a new repo from the Babylon.js 
+[Asset Host Template](https://github.com/BabylonJS/asset-host-template), 
+put the assets I'd been prototyping with in there, then changed the links 
+in my prototype code to point to my local asset host. **Fifteen minutes 
+later, my entire prototype—source code and assets—was running from my own 
+repositories.**
+1.	Now with everything moved into dev infrastructure, I set about turning
+my prototype into real production code and assets. **I refactored and
+revised to get rid of Playground-specific practices and transform my
+repos into efficient development environments where I could add features,
+take dependencies, experiment, collaborate, and iterate throughout the
+whole development process.**
+1.	Once I had something I was ready to share, I wanted to host it using 
+[GitHub Pages](https://pages.github.com/) so I could get feedback from 
+friends and colleagues. Because the Babylon.js Template Repositories are 
+set up to make this easy, **it took me five minutes, if that, to publish my 
+experience to a GitHub-hosted site I could link to.**
+1.	Next, I uploaded my game code as an 
+[NPM package](https://www.npmjs.com/) so that I could easily re-add it to 
+other projects (websites, NPM apps, etc.) just by pulling down a package. 
+Again, the Babylon.js Template repositories are set up to make this easy: 
+**within ten minutes, I had my new NPM package.**
+1.	I wanted to publish my app as a mobile game and get it listed on Google 
+Play and the App Store, so I decided to use 
+[Ionic](https://ionicframework.com/). All I had to do was create a new 
+Ionic app, add my new NPM package as a dependency, then invoke my code and 
+tell it where to display in the app. **About an hour later, my code was 
+built into a native app, ready to be shipped for mobile devices or as a 
+PWA.**
+
+![The Path: Playground to GitHub to NPM to Ionic](/img/devVignettes/fruitFalling/answer.jpg!800)
+
+And that is how I created 
+[_Fruit Fallin'_](https://syntheticmagus.github.io/fruit-falling-source/), 
+a Babylon.js app inspired by browser games from the heyday of Flash. While 
+I happened to be making a simple game, this same process can be used for 
+more complex experiences, commercial scenarios... At a high level, the 
+procedure described above can be used to take any Babylon.js idea from 
+Playground to production using the Babylon.js Template Repositories, NPM, 
+and Ionic.
+
+For a less high-level look, let's take a more step-by-step journey through 
+the process.
+
+## A More Step-by-Step Journey Through the Process
+

```

Comment:
I think a first step titled something like "Develop your idea/prototype" on the Playground, that links to the Fruit Fallin Playground and also links to the Playground doc page would be cool!

Reply:
Done. I don't think it's a separate section or step -- this Story is a journey *away* from the Playground, so I don't want to spend time showing things *in* the Playground (especially since we already have tons of docs showing that) -- but I think linking the Playground prototype in question makes a lot of sense.